inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp

```diff
@@ -228,7 +228,7 @@ TEST_P(ExecGraphTests, CheckExecGraphInfoSerialization) {
     // Create CNNNetwork from ngrpah::Function
     InferenceEngine::CNNNetwork cnnNet(function);
     InferenceEngine::CNNNetwork execGraph;
-    if (targetDevice == CommonTestUtils::DEVICE_CPU || targetDevice == CommonTestUtils::DEVICE_GPU) {
+    if (targetDevice == CommonTestUtils::DEVICE_CPU || targetDevice == CommonTestUtils::DEVICE_GPU || targetDevice == CommonTestUtils::DEVICE_MYRIAD) {
```

Comment:
could you please just remote this `if`? the body of `if` should be the same for all devices.
BTW, don't we need to enable this tests for HDDL as well?

Reply:
I'll enable these tests for the HDDL plugin after merging this PR.